### PR TITLE
release notes for March 19th 2023

### PR DESCRIPTION
**Pull request by allyford**
Includes releases from 3/5-3/19

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+
+### Release notes
+
+* Bug Fix
+  * Fixed an issue where default Linux sysctls were not applied if users specified any Linux OS config. If these sysctls were not specified, the defaults may previously have changed unintentionally: net.core.somaxconn, net.ipv4.tcp_max_syn_backlog, net.ipv4.neigh.default.gc_thresh1, net.ipv4.neigh.default.gc_thresh2, and net.ipv4.neigh.default.gc_thresh3.
+
+* Behavior Changes
+  * Podsubnet won't work in ussec/usnat. Mooncake and fairfax regions will not be affected.
+  * Addon v2 has been disabled for overlay-upgrade-data in uksouth. New clusters will install addon overlay-upgrade-data with addon v1.
+  * To enable InifinBand, set SPG and PlatformDomainCount. This setting is invalid and cannot be enabled for node pools with multile zones.
```
Comment (wangyira) [inline CHANGELOG.md:31]: small spelling issue: "multiple"

**Review by wangyira — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+  * All AKS clusters on version 1.26+ will use the latest coreDNS version [v1.10.1.](https://github.com/coredns/coredns/releases/tag/v1.10.1).
+    * For all AKS clusters on version 1.26+, coreDNS health plugin will use lameduck 5s to minimizes DNS resolution failures during coreDNS pod restart or deployment rollout. 
+    * For all AKS clusters on version 1.26+, coreDNS will use ttl 30 as default TTL for DNS records.
+* Starting with Kubernetes 1.27:
+  * The Max Surge default value will change on newly created nodepools from 1 to 10%.
+
+### Release notes
+
+* Bug Fix
+  * Fixed an issue where default Linux sysctls were not applied if users specified any Linux OS config. If these sysctls were not specified, the defaults may previously have changed unintentionally: net.core.somaxconn, net.ipv4.tcp_max_syn_backlog, net.ipv4.neigh.default.gc_thresh1, net.ipv4.neigh.default.gc_thresh2, and net.ipv4.neigh.default.gc_thresh3.
```
Comment (kaarthis) [inline CHANGELOG.md:26]: Looks pretty technical and since there are multiple sysctls any doc link to elaborate may help here ?

**Review by kaarthis — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+
+* Behavior Changes
+  * Podsubnet won't work in ussec/usnat. Mooncake and fairfax regions will not be affected.
```
[the remark this answers is not in the document]
Comment (allyford) [reply CHANGELOG.md]: removed

**Review by justindavies — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-03-19
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+ 
+* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
+* Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/en-us/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
```
Comment (phealy) [inline CHANGELOG.md:10]:
```suggestion
* Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
```

Please remove language tags from documentation links.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,41 @@
...
+  * CNI Overlay uses hostProcess containers on Windows node pools (CNS) which is not compatible with nodes using docker as the container runtime. Clusters that may be upgrading from CNI v1 using docker will break. Upgrade the container runtime to contianerd on those node pools.
```
Comment (phealy) [inline CHANGELOG.md:11]:
```suggestion
```

Overlay upgrade functionality is not yet available, no need to announce this.

**Review by phealy — CHANGES_REQUESTED** [review]
See feedback.

**Review by phealy — APPROVED** [review]

**Pull request merged**